traffic_ops/traffic_ops_golang/traffic_ops_golang.go

```diff
@@ -59,6 +59,7 @@ func main() {
 	configFileName := flag.String("cfg", "", "The config file path")
 	dbConfigFileName := flag.String("dbcfg", "", "The db config file path")
 	riakConfigFileName := flag.String("riakcfg", "", "The riak config file path")
+	useIMS := flag.Bool("use_ims", false, "Use If-Modified-Since requests")
```

Comment:
Can we make this `use-ims` instead? Hyphens are easier to type in command line flags.

Also, the new option should be added to [the Traffic Ops usage section](https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops.html#traffic-ops-golang) as well as to the usage example therein.

Reply:
Rather than being a CLI flag this should probably just be an option in TO's config file instead.

Reply:
I'm good with that too.

In that case it should be noted in the [TO configuration file section](https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops.html#cdn-conf).

Reply:
I'll get that changed.

Reply:
Done